homeassistant/components/arve/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "arve",
+  "name": "Arve",
+  "codeowners": ["@ikalnyi"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/arve",
+  "homekit": {},
+  "iot_class": "cloud_polling",
+  "requirements": ["asyncarve==0.0.4"],
```

Comment:
btw we require the code to be hosted in a public repository in GitHub or GitLab and I couldn't find it, could you please share a link?

Reply:
Hello, thanks for your comments, do you mean public repository of the python library ?

Reply:
yes exactly

Reply:
Here is the link to the public repository: https://github.com/arvetech/asyncarve

Reply:
thx!